src/IW/Sync/Search.hs

```diff
@@ -229,10 +222,9 @@ fromGitHubIssue githubIssue = do
 Parsing the URL @https://api.github.com/repos/owner/name/issues/1@ should return
 @Just (RepoOwner "owner", RepoName "name")@.
 -}
-parseUserData :: GitHub.URL -> Maybe (RepoOwner, RepoName)
-parseUserData (URL url) =
-    T.stripPrefix "https://api.github.com/repos/" url
-    >>= splitOwnerAndName
+parseUserData :: WithError m => GitHub.URL -> m (RepoOwner, RepoName)
+parseUserData (URL url) = notFoundOnNothing
```

Comment:
I guess we can provide a more verbose log in case of `Nothing` as well. As `splitOwnerAndName` is a pure function, we would know the reason for failed parsing for sure.

Reply:
@vrom911 Excuse me, but I'm not sure if I understood what you meant. I added a more verbose log message, but I'm not sure if it is helpful. Did you mean a log message for `stripPrefix`?